src/algokit/cli/deploy.py

```diff
@@ -0,0 +1,108 @@
+import logging
+import os
+from pathlib import Path
+
+import click
+
+from algokit.core.constants import DEFAULT_NETWORKS, DEPLOYER_KEY, DISPENSER_KEY, LOCALNET, MAINNET
+from algokit.core.deploy import (
+    execute_deploy_command,
+    get_genesis_network_name,
+    load_deploy_command,
+    load_deploy_config,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def extract_mnemonics(skip_mnemonics_prompts: bool, network: str) -> tuple[str | None, str | None]:  # noqa: FBT001
```

Comment:
in general, don't suppress linting or mypy errors without a good reason - usually that reason is because a call signature or whatever is dictated by external code. here it's not, you can just make `skip_mnemonics_prompts` kwargs-only.

Reply:
Done